aa_engine/aas_retrieve_inputs_part2.m

```diff
@@ -59,20 +59,23 @@
                             aas_log(aap,false,sprintf(' retrieve remote stream %s from %s:%s to %s',streamfiles(depind).streamname,streamfiles(depind).inputstream.host,logsafe_path_src,logsafe_path_dest),aap.gui_controls.colours.inputstreams);
 
                             oldpth='';
-
                             % rsync in chunks. It will then compress.
                             chunksize=64;
                             transfernow=false;
                             numtotransfer=0;
                             inps='';
+                            use_remotesymlinks = 0;
```

Comment:
I just noticed.

I think this variable should be initialized inside the for loop.

Because what if the next stream `ismodified==true`? The variable be set globally, so it would cause issues for upcoming streams I guess? What do you think? @tiborauer

Reply:
Same as for lines 72-74. I thinks 67 should be replaced with

```matlab
use_remotesymlinks = ~streamfiles(depind).ismodified && (aap.options.remotesymlinks == 1);
```